chain/cosmos/chain_node.go

```diff
@@ -1333,12 +1334,36 @@ func (tn *ChainNode) CreateNodeContainer(ctx context.Context) error {
 
 	var cmd []string
 	if chainCfg.NoHostMount {
-		cmd = []string{"sh", "-c", fmt.Sprintf("cp -r %s %s_nomnt && %s start --home %s_nomnt --x-crisis-skip-assert-invariants", tn.HomeDir(), tn.HomeDir(), chainCfg.Bin, tn.HomeDir())}
+		startCmd := fmt.Sprintf("cp -r %s %s_nomnt && %s start --home %s_nomnt --x-crisis-skip-assert-invariants", tn.HomeDir(), tn.HomeDir(), chainCfg.Bin, tn.HomeDir())
+		if len(chainCfg.AdditionalStartArgs) > 0 {
+			startCmd = fmt.Sprintf("%s %s", startCmd, chainCfg.AdditionalStartArgs)
+		}
+		cmd = []string{"sh", "-c", startCmd}
 	} else {
 		cmd = []string{chainCfg.Bin, "start", "--home", tn.HomeDir(), "--x-crisis-skip-assert-invariants"}
+		if len(chainCfg.AdditionalStartArgs) > 0 {
+			cmd = append(cmd, chainCfg.AdditionalStartArgs...)
+		}
+	}
+
+	usingPorts := nat.PortMap{}
+	for k, v := range sentryPorts {
+		usingPorts[k] = v
+	}
+
+	if tn.Index == 0 && chainCfg.HostPortOverride != nil {
```

Comment:
TODO: This has not been added to v7 interchaintest yet. Not a requirement but it is missing that feature, find to keep in v8+ though